tests/integration/test_source_git.py

```diff
@@ -45,7 +45,6 @@
 def test_basic_local_update_without_patching(
     sourcegit_and_remote,
     distgit_and_remote,
-    mock_patching,
```

Comment:
@lachmanfrantisek I'm not sure about the intention of this test. The name suggests, that this should be without patching, and so it seems to me that `mock_patching` can be removed. Am I misunderstanding something?

Reply:
You are right, it does not make sense to have it. (Maybe only the same calls with `.times(0)`..;)

Reply:
Thanks for the confirming it! :slightly_smiling_face: